layouts/joomla/form/renderfield.php

```diff
@@ -32,13 +32,12 @@
 $id    = $name . '-desc';
 
 ?>
-<div class="control-group<?php echo $class; ?>"<?php echo $rel; ?>>
+<div class="form-group<?php echo $class; ?>"<?php echo $rel; ?>>
```

Comment:
these changes are completely wrong

Reply:
Thanks for your quick review. control-group is an outdated bootstrap class that I have replaced with the currently used Bootstrap 4 class. https://getbootstrap.com/docs/4.3/components/forms/
https://bootstrapdocs.com/v2.3.2/docs/base-css.html#forms

Reply:
The problem with that change is you have not also changed the class on the inputs.

Reply:
that doesnt mean I approve of the change - just that what you have done is incomplete and requires a lot more than you have done here

Reply:
It's reverted.